nautobot/core/__init__.py

```diff
@@ -1,4 +1,4 @@
-from nautobot.core import checks
+from nautobot.core import checks  # noqa: F401
```

Comment:
As I mentioned in my general comment, I think adding `"checks"` to `__all__` would be a better fix here.